server/util/clickhouse/clickhouse.go

```diff
@@ -179,11 +188,16 @@ func (h *DBHandle) FlushInvocationStats(ctx context.Context, ti *tables.Invocati
 
 func runMigrations(gdb *gorm.DB) error {
 	log.Info("Auto-migrating clickhouse DB")
-	gdb = gdb.Set("gorm:table_options", (&Invocation{}).TableOptions())
-	if clusterOpts := (&Invocation{}).TableClusterOption(); clusterOpts != "" {
+	if clusterOpts := tableClusterOption(); clusterOpts != "" {
 		gdb = gdb.Set("gorm:table_cluster_options", clusterOpts)
 	}
-	return gdb.AutoMigrate(&Invocation{})
+	for _, t := range getAllTables() {
+		gdb = gdb.Set("gorm:table_options", t.TableOptions())
+		if err := gdb.AutoMigrate(&Invocation{}); err != nil {
```

Comment:
should this be `t` instead of `&Invocation{}`?

Reply:
Done.